packages/system-upgrade-controller/charts/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.25.0-0 < 1.32.0-0'
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.10.0-0'
+  catalog.cattle.io/release-name: system-upgrade-controller
+apiVersion: v1
+appVersion: v0.14.2
+description: General purpose controller to make system level updates to nodes.
+home: https://github.com/rancher/system-upgrade-controller
+kubeVersion: '>= 1.25.0-0'
+name: system-upgrade-controller
+sources:
+- https://github.com/rancher/charts/tree/dev-v2.10/charts/system-upgrade-controller
```

Comment:
Checking a few charts I don't see us referencing `charts` in sources and reference to `system-upgrade-controller` is covered by `home` attribute already - so suggesting to remove `sources` here.

Reply:
It was removed.